--- lire-20020214/doc/Makefile.in	Thu Mar 28 22:15:42 2002
+++ lire-20020214/doc/Makefile.in.orig	Thu Mar 28 22:18:30 2002
@@ -156,7 +156,7 @@
 
 
 docdir = $(datadir)/doc/$(PACKAGE)
-doc_DATA = user-manual.pdf user-manual.txt 	dev-manual.pdf dev-manual.txt 	faq.pdf faq.txt 	roadmap.pdf roadmap.txt
+doc_DATA = developer.html developer.txt   manual.html manual.txt roadmap.html roadmap.txt
 
 
 xml_source = roadmap.dbx user-manual.dbx dev-manual.dbx faq.dbx
